BD-4071: Clarify rate limiting details (#9338)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/rate-limiting.md

1. [**User-centric rate limiting:**](#user-centric-rate-limiting) Focuses on providing the best experience for the user.
2. [**Delivery speed rate limiting:**](#delivery-speed-rate-limiting) Takes into consideration the bandwidth of your servers.

> Braze will attempt to distribute the message sends evenly throughout the minute, but can't guarantee this. For example, if you have a campaign with a rate limit of 5,000 messages per minute, we'll attempt to distribute the 5,000 requests evenly through the minute (about 84 messages per second), but there may be some variation in the per-second rate.

Reviewer: ```suggestion
Braze will try to evenly distribute the message sends throughout the minute, but can't guarantee this. For example, if you have a campaign with a rate limit of 5,000 messages per minute, we'll try to distribute the 5,000 requests evenly through the minute (about 84 messages per second), but there may be some variation in the per-second rate.
```

#### Rate limiting and Connected Content retries

> When the [Connected Content retry][19] is turned on, Braze will retry call failures while respecting the rate limit you set for each resend. Let’s consider the scenario of 75,000 messages with a 10,000 per minute rate limit. In the first minute, the call fails or is slow and only sends 4,000 messages.

Reviewer: ```suggestion
When the [Connected Content retry][19] is turned on, Braze will retry call failures while respecting the rate limit you set for each resend. Let’s consider the scenario of sending 75,000 messages with a 10,000 per minute rate limit. Imagine that in the first minute, the call fails or is slow and only sends 4,000 messages.
```

> Instead of attempting to make up for the delay and send the remaining 4,000 messages in the second minute or add it to the 10,000 it is already set to send, Braze will move those 6,000 failed messages to the “back of the queue” and add an additional minute, if necessary, to the total minutes it would take to send your message.

Reviewer: ```suggestion
Instead of trying to make up for the delay and send the remaining 6,000 messages in the second minute or add them to the 10,000 that are already set to send, Braze will move those 6,000 messages to the “back of the queue” and add a minute, if necessary, to the total minutes it would take to send your message.
```
I don't know if I'm following this correctly 🤔 I'm not clear on the count of messages that would be failed versus sent (probably because a prior sentence says, "call fails or only sends 4,000 messages").
